crd/apis/network/v1/network.go

```diff
@@ -17,3 +17,11 @@ func DefaultNetworkIfEmpty(s string) string {
 func IsDefaultNetwork(networkName string) bool {
 	return networkName == DefaultNetworkName || networkName == DefaultPodNetworkName
 }
+
+func (n *Network) IsInUse() bool {
```

Comment:
Is is probably redundant

func InUse()

if this func is to be exported, probably should have docstring

// InUse is true if the network is currently in use.